tests/translate/get_native_namespace_test.py

```diff
@@ -57,7 +57,8 @@ def _get_expected_namespace(constructor_name: str) -> Any | None:  # noqa: PLR09
 
 def test_native_namespace_frame(constructor: Constructor) -> None:
     constructor_name = constructor.__name__
-    if constructor_name == "pyspark_lazy_constructor":
+
+    if constructor_name == "pyspark_constructor":
```

Comment:
i was getting this to be `_constructor` when running locally - have addressed separately anyway